microservice_templates/microservice_node_template/supervisor/_MICROSERVICE_NODE_TEMPLATE_.conf

```diff
@@ -0,0 +1,3 @@
+[program:_MICROSERVICE_NODE_TEMPLATE_]
+directory=/opt/_MICROSERVICE_NODE_TEMPLATE_/src
+command=nodejs _MICROSERVICE_NODE_TEMPLATE_.js
```

Comment:
Afer checking:
Changing this line to `npm start` makes doesn't comply well with supervisor. Running `supervisorctl stop myservice` stop npm process but application keeps working. Also, if by any means npm process fails, it won't be restarted.